setup-local/src/artifacts.js

```diff
@@ -0,0 +1,27 @@
+const artifact = require('@actions/artifact');
+const core = require('@actions/core');
+
+class ArtifactsManager {
```

Comment:
as a good practice, shouldn't we name similar to class name?

Reply:
ah yes. will change that.

Reply:
done

Reply:
resolving this.